Core/ContentBlockerConfigurationUserDefaults.swift

```diff
@@ -66,10 +72,15 @@ public class ContentBlockerProtectionUserDefaults: ContentBlockerProtectionStore
     public func isTempUnprotected(domain: String?) -> Bool {
         guard let domain = domain else { return false }
         
+        if tempUnprotectedDomains == nil {
+            tempUnprotectedDomains = FileStore().loadAsArray(forConfiguration: .temporaryUnprotectedSites)
+                .filter { !$0.trimWhitespace().isEmpty }
```

Comment:
@bwaresiak / @SlayterDev 

I know this is merged but why wasn’t a lazy var good enough here? 

The comment says that var should be constrained to main thread but nothing is constraining this assignment to the main thread.

Reply:
This is used from UI only (at the moment) so I just put a comment there to remember when we refactor it (which we will have to do).